spec/bundler/friendly_errors_spec.rb

```diff
@@ -3,6 +3,32 @@
 require "bundler/friendly_errors"
 
 describe Bundler, "friendly errors" do
+  context "with invalid YAML in .gemrc" do
+    before do
+      File.open(Gem.configuration.config_file_name, 'w') do |f|
```

Comment:
this is in a tmpdir during the tests, right? If not, that'll clobber my config :(